src/Sarif.UnitTests/Core/StackTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Linq;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    [TestClass]
+    public class StackTests
+    {
+        [TestMethod]
+        public void Stack_CreateFromStackTrace()
+        {
+            var dotNetStack = new StackTrace();
+            Stack stack = Stack.Create(dotNetStack);
+
+            // The .NET StackTrace.ToString() override must preserve a trailing NewLine
+            // for compatibility reasons. We do not retain this behavior in ToString()
+            // but provide a facility for adding the trailing NewLine
+            Assert.AreEqual(dotNetStack.ToString(), stack.ToString(StackFormat.TrailingNewLine));
```

Comment:
There's nothing wrong with the Assert API _per se_. It's just that FluentAssertions has two features which I consider compelling. One is objective, the other subjective:

1. Objective: FluentAssertions has a far larger set of assertions than the Assert API. Take a look at its [wiki page](https://github.com/dennisdoomen/fluentassertions/wiki). To really appreciate it, keep scrolling down. And down. It's amazing how much good stuff is in there.

2. Subjective: Reasonable people can disagree whether `stacks.Count.Should().Be(2)` is easier to read than `Assert.AreEqual(2, stacks.Count)`. But when you get down to something like this, IMO the case for the fluent syntax becomes much more compelling:

```C#
Action act = () => subject.Foo(null)); 
act
  .ShouldThrow<ArgumentNullException>()
  .WithMessage("?did*");
```

My 2c